atc/builds/planner.go

```diff
@@ -95,13 +95,13 @@ func (visitor *planVisitor) VisitGet(step *atc.GetStep) error {
 	visitor.plan = visitor.planFactory.NewPlan(atc.GetPlan{
 		Name: step.Name,
 
-		Type:     resource.Type,
-		Resource: resourceName,
-		Source:   resource.Source,
-		Params:   step.Params,
-		Version:  &version,
-		Tags:     step.Tags,
-		Timeout:  step.Timeout,
+		Type:                 resource.Type,
```

Comment:
Would you mind running `go fmt` to get rid of these? Just aiming to preserve `git blame` integrity since these don't need to change anymore.

Reply:
@vito  I usually run `go fmt` before commit, sorry for missing this file.

I use IntelliJ IDEA. The IDEA has a feature of "format project" that can `go fmt` all files of the project, which is very convenient. But unfortunately I cannot use the feature, I tried, "format project" would cause many other files formating that I didn't touch, meaning that, there are many files in the project not `go fmt`ed. For this reason, I have to format changed files one by one, then potentially, missing some files.

Would you mind me to submit a PR to correct all missing formatted files? And afterwards, we may consider to add a format check in CI pipeline. The logic should be simple:

```
for each files in commit
     out_file = `go fmt original_file`
     if out_file != original_file
          fail
     end
end
```

which will make sure all changes well `go fmt`ed.